### PR TITLE
Feature/quasar ui test

### PR DESCRIPTION
**Pull request by PiddePannkauga**
QUASAR will now be used as component library!

HOW TO TEST:

Build docker. 
Press "Sign up" in top right and create an account. Check firebase that account was created.
Press "Login" in top right and login with an existing account. Check network in dev tools and see if something is returned.

### CODE REVIEW

File: client/src/views/Home.vue [hunk 1]
```diff
@@ -1,19 +1,12 @@
 <template>
-  <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-    <HelloWorld msg="Wollolooo
-    "/>
-  </div>
+  <div>OOOGABOOGA</div>
```
Comment (Kalk88) [inline client/src/views/Home.vue]: I don't understand this.

File: client/src/main.js [hunk 2]
```diff
@@ -2,11 +2,42 @@ import Vue from 'vue'
...
+import {
+  InMemoryCache
+} from 'apollo-cache-inmemory'
+
+
+Vue.use(VueApollo)
+
+const httpLink = createHttpLink({
+  // You should use an absolute URL here
+  uri: 'http://localhost:9090/graph',
```
Comment (Kalk88) [inline client/src/main.js]: This needs to be set dynamically depending on prod or dev.

File: client/src/views/Home.vue (hunk 1 above)
Comment (PiddePannkauga) [reply client/src/views/Home.vue]: Placeholder content

File: client/src/main.js [hunk 3]
```diff
@@ -2,11 +2,43 @@ import Vue from 'vue'
...
+} from 'apollo-cache-inmemory'
+
+
+Vue.use(VueApollo)
+
+const httpLink = createHttpLink({
+  // You should use an absolute URL here
+  uri: process.env.VUE_APP_API_ENDPOINT_URL,
+})
+console.log(process.env.VUE_APP_API_ENDPOINT_URL);
```
Comment (Kalk88) [inline client/src/main.js]: remove this

**Pull request merged**